lib/CMakeLists.txt

```diff
@@ -0,0 +1,31 @@
+### lib/CMakeLists.txt ###
+#
+#
+
+## Platform and system libraries
+include(platform_libs.cmake)
```

Comment:
Agreed this is out of place, I'll change it to `PlatformLibs.cmake` before merging.